python-flask/python-flask-example.py

```diff
@@ -0,0 +1,35 @@
+from flask import Flask, json, request
+import base64
+import pprint
+import datetime
+app = Flask(__name__)
+
+@app.route('/', defaults={'path': ''}, methods=['POST'])
+@app.route('/<path:path>', methods=['POST'])
+def log_payload(path):
+    # Get request payload
+    payload = request.json
+
+    # Ensure that payload is a dictionary
+    if not isinstance(payload, dict):
+        payload = {'raw_data':payload}
+
+    # Get message data
+    timestamp = datetime.datetime.now()
+    message = payload.get('message',{})
+    if message:
+        # Get device ID
+        id = message.get('attributes',{}).get('deviceId','')
+        # Get base64 encoded data payload
+        data = base64.b64decode(message.get('data',''))
+        print("----- Got message from {} ({}) -------".format(id, timestamp))
+        print("Message path: /{}".format(path))
```

Comment:
Perhaps we should move this out of the if/else? Or remove it? Not sure what purpose it serves?

Reply:
Yes, just had it there for debugging purposes, removed, flask prints it anyway..